test/suite/__snapshots__/extension.test.js.snap

```diff
@@ -25,11 +25,11 @@ export { options };
 
 exports[`Extension Test Suite setting "prettierLast" makes prettier run last 1`] = `
 "const options = {
-    first: "hello, world",
+    first: 'hello, world',
```

Comment:
This indicates that there is a bug as the snapshots shouldn't change between versions